src/components/ContentWidgets/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+import './Accordion.css';
+
+/**
+ * Accordion is a collapsible list of titles and coorisponding content
+ * 
+ * Last Modified
+ * Jennifer Tao
+ * July 21, 2019
+ */
+export const Accordion: React.FC<ContentSingularData> = ({
+    accordion_content
+}) => {
+    if (!accordion_content || accordion_content.length) {
+        return <></>
+    }
+
+    const collapsePanel = (e: any) => {
```

Comment:
I would suggest instead having this be a state, rather than you directly modifying the `className`. I don't think people generally do that and we've primarily been using state and having the class name adjust within the return.

Reply:
So it would look something like maybe an array of collapsed states and then you would `collapsePanel(index)` and then down below you would probably have 

```
className={`${collapsed[i] ? "" : "active-accordion"}`}
```

Reply:
I created the component `AccordionPage` and set the collapse state, `isActive` as its state using `useState`.